CHANGELOG.md

```diff
@@ -118,6 +118,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - MapboxOSRM directions was parsing geometry to \[lat, lon\]
 - README not valid on PyPI, wasn't rendered
 - add to PyPI and Conda
+- Google router's `duration` and `distance` attributes not being calculated correctly in single route response ([#107](https://github.com/gis-ops/routingpy/issues/107))
+
```

Comment:
This is the oldest section, v0.2. Can you put at the top under `Unreleased` in a new `Fixed` section?

Reply:
Sorry, I missed your message, updating it ASAP!

Reply:
But why both now? :sweat_smile: It wasn't released yet, so it only belongs to the Unreleased section

Reply:
> Unreleased is available in Github's master branch, but not on PyPI.

It wasn't merged to master at the time :smile: